File: wanggsx_wifi_transfer/src/main/java/com/wanggsx/wanggsx_wifi_transfer/FileTransfer.java
package com.wanggsx.wanggsx_wifi_transfer;

import java.io.Serializable;

public class FileTransfer implements Serializable {

    //文件路径
    private String filePath;

    //文件大小
    private long fileLength;

    //MD5码
    private String md5;

}

File: wifitransfer/src/main/java/com/wanggsx/wifitransfer/FileTransfer.kt
package com.wanggsx.wifitransfer

import java.io.Serializable

class FileTransfer(//文件路径
        var filePath: String?, //文件大小
        var fileLength: Long) : Serializable {

    //MD5码
    var md5: String? = null

    override fun toString(): String {
        return "FileTransfer{" +
                "filePath='" + filePath + '\''.toString() +
                ", fileLength=" + fileLength +
                ", md5='" + md5 + '\''.toString() +
                '}'.toString()
    }

}